main.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"github.com/astaxie/beego"
+	_ "goMarketer/routers"
```

Comment:
How about move internal project packages up above 3rd party packages and separate them by blank line?

Ref : [Our Go convention](https://github.com/nimblehq/compass/blob/1f99862fb1c202472e9346d1b85a591e993fd062/_docs/development/code-conventions/go.md)

Since our Go convention is not done yet, but all of that conventions come from our vote and [Effective Go](https://github.com/golovers/effective-go#references). I think we can follow what we have right now.

What do you think? cr. @olivierobert

Reply:
Fixed that in https://github.com/andyduong1920/go_marketer/pull/1/commits/37ca365970a8d680f8ee429a370a5a800dbcb02e @gutakk @olivierobert   🚀